syntax = "proto3";

import "ClassInfo.proto";

package class_system;

message Request {
  string key                           = 1;
  bool   request_class_info            = 2;
  bool   request_sentences             = 3;

  // ClassInfo to be saved
  optional ClassInfo modified_class_info = 4;
}
